Bottom: e1fc61872332f367af325d75e1381c85c7b6e42c
Top:    5afc89e1a868b28d319b8b0c89f8a27bcd122568
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 02:31:40 +0100

Refresh of off-to-find-burried-treasure-o

---

diff --git a/src/draw.c b/src/draw.c
index 7881ef9..afa0058 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -894,7 +894,7 @@ poly_callback (const BoxType * b, void *cl)
     return 0;
 
   /* Re-use HOLEFLAG to cut out islands */
-  if (TEST_FLAG (HOLEFLAG, Polygon))
+  if (TEST_FLAG (HOLEFLAG, polygon))
     return 0;
 
   if (TEST_FLAG (SELECTEDFLAG, polygon))
